src/constants.lua

```diff
@@ -9,6 +9,8 @@ constants.defaultundernameLimit = 10
 constants.totalTokenSupply = 1000000000 * 1000000 -- 1 billion tokens
 constants.MIN_EXPEDITED_WITHDRAWAL_PENALTY_RATE = 0.10 -- the minimum penalty rate for an expedited withdrawal (10% of the amount being withdrawn)
 constants.MAX_EXPEDITED_WITHDRAWAL_PENALTY_RATE = 0.50 -- the maximum penalty rate for an expedited withdrawal (50% of the amount being withdrawn)
+constants.oneTokenInBaseUnits = 1000000
```

Comment:
some bad copy pasta left us in a state of inconsistent formatting.

let's use standard format and rename this `mIOPerIO` or something specific to the name of base unit relative to IO